projects/canopy/src/lib/pagination/docs/guide.stories.mdx

```diff
@@ -0,0 +1,70 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import Feedback from '../../docs/common/feedback.md';
+
+<Meta title="Components/Pagination/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Pagination
+
+<p class="standfirst">Controls a paginated list of items</p>
+
+## Usage
+
+Import the pagination module in your application:
+
+```typescript
+@NgModule({
+  ...
+  imports: [ ..., LgPaginationModule ],
+});
+```
+
+In your HTML
+```html
+<lg-pagination
+  [totalItems]="allItems.length"
+  [itemsPerPage]="itemsPerPage"
+  (pageChanged)="onPageChanged($event)"
+>
+</lg-pagination>
+```
+
+And in your code
+```typescript
+
+import { PageData } from '@legal-and-general/canopy';
+
+...
+
+  export class PagingExampleComponent implements OnInit{
+    allItems: = [];
+    pagedItems: User[] = [];
+    itemsPerPage = 10;
+
+    ngOnInit(): void {
+      this.allItems = this.someService.getData();
+
+      // Create the first page of data on init
+      this.pagedItems = this.getPagedData(0, this.itemsPerPage - 1);
+    }
+
+    onPageChanged(pageData: PageData): void {
+      // And then respond to onPageChangeEvents to update the paged data
+      this.pagedItems = this.getPagedData(pageData.startIndex, pageData.endIndex)
+    }
+
+    private getPagedData(startIndex: number, endIndex: number): unknown[] {
+      // Important! Be careful if using array.prototype.splice as it returns
+      // items from startIndex, up to but not including endIndex.
+      return this.allItems.slice(startIndex, endIndex + 1);
+    }
+  }
+}
+```
+
+## Additional development details
+
+The component will automatically hide itself the the total number of items is les than or equal to the `itemsPerPage` input (i.e. There is only 1 page of data)
```

Comment:
```suggestion
The component will automatically hide itself the the total number of items is less than or equal to the `itemsPerPage` input (i.e. There is only 1 page of data)
```

Reply:
Done, Also included som additional requests from @elenagarrone . To the text is now 
![Screenshot 2023-02-27 at 12 52 45](https://user-images.githubusercontent.com/22469863/221568754-f1d88a0d-3529-4d32-833d-9075954c13f2.png)